packages/diagnostic/tests/fixtures/app/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
inherit from project base `tsconfig.json`

Reply:
This is fixture's tsconfig.json. Let's keep it independent.